NEWS

```diff
@@ -33,6 +33,23 @@ CHANGES WITH 255 in spe:
           by default when combined with --scope, will be changed in a future
           release to be enabled by default.
 
+        Device Management:
+
+        * udev will now create symlinks to loopback block devices in the
+          /dev/disk/by-ref/ directory that are based on the .lo_file_name
```

Comment:
hmm, /dev/disk/ is a bit generic for some super-vague term just "ref". this names should really carry some reference to "loop".

Maybe /dev/disk/by-loop-ref/? Or /dev/disk/loop/by-ref/? 

Kinda leaning forwards the former, i.e. /dev/disk/by-loop-ref + /dev/disk/by-loop-inode?

Reply:
The switch to losetup is named --loop-ref=, hence I theink /dev/disk/by-loop-ref/ would be the most obvious choice, to directly reuse that naming in the symlink.